tests/unit/lib/build_module/test_build_strategy.py

```diff
@@ -156,6 +157,7 @@ def test_build_layers_and_functions(self, mock_copy_tree, mock_path):
                     self.function_build_definition2.get_handler_name(),
                     ANY,
                     self.function_build_definition2.metadata,
+                    self.function_build_definition1.env_vars,
```

Comment:
double check here, why it is `definition1` here?

Reply:
good catch! because they're both mock results, that's why tests don't fail